Lib/idlelib/idle_test/test_tooltip.py

```diff
@@ -1,27 +1,31 @@
+from functools import wraps
 from idlelib.tooltip import TooltipBase, Hovertip
 from test.support import requires
-requires('gui')
-
-from functools import wraps
 import time
 from tkinter import Button, Tk, Toplevel
 import unittest
 
 
+requires('gui')
+
+
 def setUpModule():
     global root
     root = Tk()
 
+
 def root_update():
     global root
     root.update()
 
+
```

Comment:
idlelib is not consitently double spaced around functions.  To me, it is less needed than for classes.  And PEP8 has this, which I think applies here: "Blank lines may be omitted between a bunch of related one-liners".

That aside, root_update just adds call overhead.  I am replacing it.